data/locale/fr-FR.ini

```diff
@@ -1,37 +1,55 @@
 NDIPlugin.Default="Par défaut"
-NDIPlugin.NDISourceName="Source NDI™"
+NDIPlugin.NDISourceName="Source NDI®"
 NDIPlugin.SourceProps.SourceName="Source"
-NDIPlugin.SourceProps.Bandwidth="Bandwidth"
+NDIPlugin.SourceProps.Bandwidth="Bande passante"
+NDIPlugin.SourceProps.Bandwidth.Highest="Le plus haut"
+NDIPlugin.SourceProps.Bandwidth.Lowest="Le plus bas"
```

Comment:
Grammatically incorrect. "Bande passante" is feminine in french. so that would be "La plus élevée"
"La plus basse"
> Updated in crowdin